distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/restito/ChunkedContent.java

```diff
@@ -130,6 +131,12 @@ public Response apply(Response response) {
         private Response respond(Response response) {
             for (char c : responseMessage.toCharArray()) {
                 try {
+                    String log = String.format("[2287] Sending Character: [%s] at %s%n%n",
+                            String.valueOf(c),
+                            DateTime.now()
+                                    .toString());
+                    System.out.println(log);
```

Comment:
For some reason the logger statements were not showing up. Do tests currently default to debug or do I need to pass a parameter?